backend/.env.template

```diff
@@ -23,3 +23,5 @@ AWS_SECRET_ACCESS_KEY=
 AWS_ENDPOINT=
 AWS_REGION=
 AWS_BUCKET=
+IMGPROXY_KEY=21b2b7b270d5fd48f02491e520fc89b8f31bab66ef2a00b3f254ce3f02c6fae6407345f3b394580ffa751973f624ce0740566ff39960f75da6ec9f45dfc7b9c2
+IMGPROXY_SALT=8144a980d252b21aa5ea8c82379f3b7948dfb6373f60a2f633de40d64be8a326de46490aef2f4c2ed4278b76cf1309fd004338026a7737c8ca805415c38605a4
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-30T21:32:00Z" title="Monday, March 30th 2020, 11:32:00 pm +02:00">Mar 30, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -23,3 +23,5 @@ AWS_SECRET_ACCESS_KEY=
 AWS_ENDPOINT=
 AWS_REGION=
 AWS_BUCKET=
+IMGPROXY_KEY=21b2b7b270d5fd48f02491e520fc89b8f31bab66ef2a00b3f254ce3f02c6fae6407345f3b394580ffa751973f624ce0740566ff39960f75da6ec9f45dfc7b9c2
+IMGPROXY_SALT=8144a980d252b21aa5ea8c82379f3b7948dfb6373f60a2f633de40d64be8a326de46490aef2f4c2ed4278b76cf1309fd004338026a7737c8ca805415c38605a4
```


This secret will be different in production

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-01T06:32:56Z" title="Wednesday, April 1st 2020, 8:32:56 am +02:00">Apr 1, 2020</time>_

---


I hope so! hehe